Documentation/help_files/medical_dashboard_help.php

```diff
@@ -0,0 +1,235 @@
+                    
+                    <p><?php echo xlt("Enable only one type of portal by checking the relevant check-box");?>.
+                    
+                    <p><i class="fa fa-exclamation-triangle oe-text-red"  aria-hidden="true"></i> <strong><?php echo xlt("You will need Administrator privileges to enable the patient portal");?>.</strong>
+                    
+                    <p><i class="fa fa-exclamation-circle oe-text-orange"  aria-hidden="true"></i> <?php echo xlt("If the patient has not authorized patient portal then a Patient has not authorized the Patient Portal message will be shown here");?>.
+                    
+                    <p><?php echo xlt("To authorize the patient portal for the patient go to Dashboard > Demographics > Edit > Choices and select Yes in Allow Patient Portal drop-down box and Save");?>.
+                    <button type="button" class="btn btn-default btn-sm oe-no-float"><?php echo xlt("Edit"); ?></button>
+                    <button type="button" class="btn btn-default btn-save btn-sm oe-no-float"><?php echo xlt("Save"); ?></button>
+                    
+                    <p><?php echo xlt("If the Online Patient portal is enabled there will be either a button that says Create Online Portal Credentials provided the patient has given permission to access the online patient portal or a message that says Patient has not authorized the Patient Portal");?>.
+                    <button type="button" class="btn btn-default btn-save btn-sm oe-no-float"><?php echo xlt("Create Online Portal Credentials"); ?></button>
+                                        
+                    <p><?php echo xlt("If the Offsite Patient portal is enabled there will be either a button that says Create Offsite Portal Credentials provided the patient has given permission to access the online patient portal or a message that says Patient has not authorized the Patient Portal");?>.
+                    <button type="button" class="btn btn-default btn-save btn-sm oe-no-float"><?php echo xlt("Create Offsite Portal Credentials"); ?></button>
+                    
+                    <p><?php echo xlt("Clicking on the Create Online/Offsite Portal Credentials button will generate a username and password for the patient that has to be given to the patient");?>.
+                                        
+                    <p><?php echo xlt("These credentials will be used by the patient to login to the patient portal for the first time");?>.
+                    
+                    <p><?php echo xlt("The patient will have to change their credentials at the first login");?>.
+                    
+                    <p><?php echo xlt("If the Online/Offsite Portal Credentials has already been set the button will change to ");?>
```

Comment:
Removing the trailing space from within this constant

Reply:
Done